int16[1] addresses
---
marvelmind_nav/hedge_pos_a[1] positions